src/app/leverage/components/stats.tsx

```diff
@@ -65,18 +62,7 @@ function StatsItem(props: StatsItemProps) {
         <div className='text-ic-gray-100 text-xs font-normal'>24h Change</div>
         {change24h.length > 0 ? (
           <div className='flex flex-row items-center gap-1'>
-            <div>
-              <svg
-                className={clsx(iconColor, iconScale)}
-                xmlns='http://www.w3.org/2000/svg'
-                width='8'
-                height='6'
-                viewBox='0 0 8 6'
-                fill='none'
-              >
-                <path d='M0.666667 5.86914L4 2.26083L7.33333 5.86914L8 5.14748L4 0.817505L6.24448e-08 5.14748L0.666667 5.86914Z' />
-              </svg>
-            </div>
+            <ChevronUpIcon className={clsx(iconColor, iconScale)} />
```

Comment:
just fyi: we have another function now called `cn` which also makes sure, that the classes are merged in correct order, while using clsx too.